lib/plug/static.ex

```diff
@@ -112,43 +112,43 @@ defmodule Plug.Static do
   end
 
   def init(opts) do
-    at     = Keyword.fetch!(opts, :at)
-    from   = Keyword.fetch!(opts, :from)
-    gzip   = Keyword.get(opts, :gzip, false)
-    brotli = Keyword.get(opts, :brotli, false)
-    only   = Keyword.get(opts, :only, [])
+    at = Keyword.fetch!(opts, :at)
+    from = Keyword.fetch!(opts, :from)
+    gzip? = Keyword.get(opts, :gzip, false)
+    brotli? = Keyword.get(opts, :brotli, false)
+    only = Keyword.get(opts, :only, [])
     prefix = Keyword.get(opts, :only_matching, [])
```

Comment:
No strong opinion here, but I tend to align assignments myself (as well as `->` clauses in cases and conds) because it reads more like a "table". However, diffs definitely suffer from this.